webrtc.html

```diff
@@ -3840,6 +3840,22 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <ol>
                 <li>
                   <p>Let <var>connection</var> be the
+                  <code><a>RTCPeerConnection</a></code> object on which this
+                  method was invoked.</p>
+                </li>
+                <li>
+                  <p>Let <var>track</var> be the <code>
+                  <a>MediaStreamTrack</a></code> object indicated by the
+                  method's first argument.</p>
+                </li>
+                <li>
+                  <p>Let <var>streams</var> be a list of <code>
+                  <a>MediaStream</a></code> objects constructed from the
+                  method's remaining arguments; if the method was called with a
+                  single argument the result is an empty list.</p>
```

Comment:
What result? Maybe: *", or an empty list if the method was called with a single argument"* ?